front/config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "apiUrl": "http://localhost:4400"
+  "apiUrl": "http://aelve.com:4800"
```

Comment:
The apiUrl was changed intentionally since we have docker to build backend on local machine. If you want use remote please dont commit it.

Reply:
done